distributed-job-manager/src/test/java/ru/fix/distributed/job/manager/assertion/RetryAssert.java

```diff
@@ -0,0 +1,186 @@
+package ru.fix.distributed.job.manager.assertion;
+
+
+import org.junit.jupiter.api.Assertions;
+
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
```

Comment:
RetryAssert was created because author did not know about /
awaitility. https://github.com/awaitility/awaitility. 
Could you please clarify, can we use well known library instead of custom asserting bicycle?

Reply:
yes we can, i will fix it soon

Reply:
fixed